library/cwm/src/lib/cwm/widget.rb

```diff
@@ -258,6 +258,60 @@ def validate_wrapper(_widget_id, _event)
     end
   end
 
+  # Wrapper when combination of old hash based CWM definition needed to be used
+  # with new object based one. Useful e.g. when content is provided by other
+  # module like CWMFirewallInterfaces.CreateOpenFirewallWidget
+  # @note it does not support common methods and helpers from abstract widget
```

Comment:
Ugh! So many `unsupported` exceptions. I suspect we are going to be arrested for violating the [law of Ms Liskov](https://en.wikipedia.org/wiki/Liskov_substitution_principle).